packages/installation-tests/esm-playwright-test.mjs

```diff
@@ -14,11 +14,21 @@
  * limitations under the License.
  */
 
-import { chromium, firefox, webkit, selectors, devices, errors, test } from '@playwright/test';
+import { chromium, firefox, webkit, selectors, devices, errors, test, expect } from '@playwright/test';
```

Comment:
This test needed to be stronger. Now i checks that `expect` is also exported (it wasn't) and that both `test` and `expect` are functions.

Reply:
We could also add `expect(1).toBe(1)` to be on the safe side. Since this call is lightweight it makes probably sense to add it, wdyt?